src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
I build it on top of  @dagood work with #646, it does not allow names other than project.json. I agree it's crappy to see another project.json file being maintained without great reason. But I wanted to minimize the build script changes,  hence minimize the risk of breaking other existing scenarios.